src/library/components/stem-finder-result.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react'
-var Component = require('../helpers/component')
+import Component from '../helpers/component'
 
-var ResourceLightbox = require('./resource-lightbox')
-var ResourceType = require('./resource-type')
-var GradeLevels = require('./grade-levels')
-var Lightbox = require('../helpers/lightbox')
+import ResourceLightbox from './resource-lightbox'
+import ResourceType from './resource-type'
+import GradeLevels from './grade-levels'
+import Lightbox from '../helpers/lightbox'
+import portalObjectHelpers from '../helpers/portal-object-helpers'
 
-var div = React.DOM.div
-var img = React.DOM.img
-var a = React.DOM.a
-var i = React.DOM.i
+const div = React.DOM.div
+const img = React.DOM.img
+const a = React.DOM.a
+const i = React.DOM.i
```

Comment:
If all of the elements have been switched to JSX, then I believe you can remove these.